frontend/apps/app/components/shared/BranchCombobox/BranchCombobox.tsx

```diff
@@ -0,0 +1,173 @@
+'use client'
+
+import { ChevronDown, Code, GitBranch, Search } from '@liam-hq/ui'
+import clsx from 'clsx'
+import { Command } from 'cmdk'
```

Comment:
Using cmdk is make sense. Excellent!!
Eventually I'd like to export from the ui package, but for now I'll leave it as is.